### DISCUSSION TITLE
Third-party library

### DISCUSSION THREAD

**Discussion by IPCamTeam** [AxisCommunications/discussions#53] [ACAP]
Hi,

We are trying to build Acap application with c++ in which we will use third party library. We have source code of  the library and  CmakeLists.txt ,but we don't know how to build it and after that how to reference it from our project. Could you please help us with some resources or examples ?

Thank you in advance

**Comment by mattias-kindborg-at-work** [comment]
Hi @IPCamTeam.

Are you using [ACAP Native SDK](https://github.com/AxisCommunications/acap-native-sdk/)?

**Reply by IPCamTeam** [reply]
Hi,
No ,we are using ACAP SDK VERSION 3.5

**Reply by mattias-kindborg-at-work** [reply]
We've got some code relating to storage [here](https://www.axis.com/techsup/developer_doc/acap3/3.5/api/axstorage/html/ax_storage_example_8c-example.html). Does that help?
